backend/src/interfaces/IService.ts

```diff
@@ -25,4 +26,25 @@ export interface IServiceDto {
   sms_description?: string;
   blocked?: boolean;
   hidden?: boolean;
+  employee_ids?: string[];
 }
+
+interface IDurationsPrices {
+  id?: string;    //  for updating a series of services
+  duration: number;
+  price: string;
+}
+
+export interface IServicesDto {
```

Comment:
What are some of the use cases for updating multiple services at once? This interface looks like a repeat of `IServiceDto` and is a burden to maintain.

Reply:
this interface is to define necessary fields for creating a service, any service info unrelated with the id should satisfy this interface

Reply:
What is the difference between `IServicesDto` and `IServiceDto`?

Reply:
`IServiceDto` has been changed into `IServicesDto`, the later one contains all neccessary information to create a group of services.